Marlin/src/pins/stm32h7/pins_BTT_SKR_SE_BX_common.h

```diff
@@ -53,13 +65,13 @@
 #define POWER_MONITOR_PIN                   PB0
 #define RPI_POWER_PIN                       PE5
 
-#define SERVO0_PIN                          PA2
+#define SERVO0_PIN                          PA2   // PROBE
 
 //
 // Z Probe (when not Z_MIN_PIN)
 //
 #ifndef Z_MIN_PROBE_PIN
-  #define Z_MIN_PROBE_PIN                   PH2   // Probe
+  #define Z_MIN_PROBE_PIN                   PH2   // PROBE
```

Comment:
Why do both PA2 and PH2 have `PROBE` comments?

I'm not sure exactly how this comment is used or if mentioning it twice makes sense. The only other board I see with it twice is the BTT_MANTA_M8P_V2_0.

Presuming these are just informational for readers of the file, then more verbose comments about having two choices could make sense.

Reply:
They are pulled from silkscreen labels on the board.